pkg/ossm/bug_multiple_smcp.go

```diff
@@ -10,41 +10,44 @@ import (
 
 func cleanupMultipleSMCP() {
 	util.Log.Info("Delete the Multiple CP")
-	// util.KubeDeleteContents(meshNamespace, smmr)
-	// util.KubeDeleteContents(meshNamespace, util.RunTemplate(smcpV23_template, smcp))
+	util.KubeDeleteContents(meshNS2, smmr)
+	util.KubeDeleteContents(meshNS2, util.RunTemplate(smcpV23_template, smcp))
 	time.Sleep(time.Duration(40) * time.Second)
```

Comment:
This sleep isn't necessary. You can immediately delete the other SMCP and then sleep only once. Or not sleep at all, because the `oc delete` command blocks until the object is fully deleted, which means that sleep after a delete is not necessary at all.